vignettes/adrs.Rmd

```diff
@@ -13,6 +13,32 @@ knitr::opts_chunk$set(
   collapse = TRUE,
   comment = "#>"
 )
+link <- function(text, url) {
+  return(
+    paste0(
+      "[", text, "]",
+      "(", url, ")"
+    )
+  )
+}
+dyn_link <- function(text,
+                     base_url,
+                     relative_url = "",
+                     # Change to TRUE when admiral adopts multiversion docs
+                     is_multiversion = FALSE,
+                     multiversion_default_ref = "main") {
+  url <- paste(base_url, relative_url, sep = "/")
+  if (is_multiversion) {
+    url <- paste(
+      base_url,
+      Sys.getenv("BRANCH_NAME", multiversion_default_ref),
+      sep = "/"
```

Comment:
`relative_url,` is missing here (and in all the other copies of the code). That's the reason why the link check fails.

Reply:
thanks - i'll update here and if that works we'll need to go back to {admiral} and do same in any other places for now before we move this out to {admiraldev}

Reply:
@bundfussr doesn't look to have worked either with the links check. shout if you spot anything i'm missing here, otherwise i'll take a deeper look tomorrow. i've been coming back and forth this one in between other bits today so not given it focus.

Reply:
In the html version it is working now. Unfortunately, the link check checks the Rmd file (because the html is not in the repository). I would suggest to add "https://github.com/pharmaverse/admiralonco/blob" to the `.lycheeignore` file.